public/app/plugins/datasource/grafana-azure-monitor-datasource/components/deprecated/types/query.ts

```diff
@@ -0,0 +1,60 @@
+import { DataQuery } from '@grafana/data';
+
+import { AzureLogsQuery, AzureMetricQuery, AzureQueryType, AzureResourceGraphQuery } from '../../../types';
+import { GrafanaTemplateVariableQuery } from '../../../types/templateVariables';
+
+export enum DeprecatedAzureQueryType {
+  ApplicationInsights = 'Application Insights',
+  InsightsAnalytics = 'Insights Analytics',
+}
+
+export interface AzureMonitorQuery extends DataQuery {
```

Comment:
Should this be in a deprecated folder and if so should it be renamed to have deprecated in the title like the others?